### PR TITLE
Handle Meta

### PR DESCRIPTION
**Pull request by dinithiW**
Issues a warning when a meta section is encountered in a WDL document.
Added new test case

### CODE REVIEW

File: wdl2cwl/main.py [hunk 1]
```diff
@@ -426,6 +426,9 @@ def convert(workflow: str) -> str:
     if ast.task_parameter_meta_check:
         print("----WARNING: SKIPPING PARAMETER_META----")
 
+    if ast.task_meta_check:
+        print("----WARNING: SKIPPING META----")
+
```
Comment (mr-c) [inline wdl2cwl/main.py:431]: Is there a test that confirms this warning is printed?

**Review by mr-c — CHANGES_REQUESTED** [review]
Looks good! Needs two specific tests to confirm that warning is printed when it should be; and not printed when it shouldn't

File: wdl2cwl/main.py (hunk 1 above)
Comment (dinithiW) [reply wdl2cwl/main.py:431]: I'm not sure why UmiCorrection.wdl won't work for this. It has a meta section.

Comment (mr-c) [reply wdl2cwl/main.py:431]: Use https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-capsys to check that the warning is printed

**Comment by dinithiW** [comment]
```
wdl2cwl/tests/test_cwl.py: note: In function "test_meta":
  wdl2cwl/tests/test_cwl.py:13: error: Missing type parameters for generic type
  "CaptureFixture"  [type-arg]
      def test_meta(capsys: pytest.CaptureFixture) -> None:
      ^
  Found 1 error in 1 file (checked 4 source files)
```
Is the type incorrect?

**Comment by mr-c** [comment]
> ```
> wdl2cwl/tests/test_cwl.py: note: In function "test_meta":
>   wdl2cwl/tests/test_cwl.py:13: error: Missing type parameters for generic type
>   "CaptureFixture"  [type-arg]
>       def test_meta(capsys: pytest.CaptureFixture) -> None:
>       ^
>   Found 1 error in 1 file (checked 4 source files)
> ```
> Is the type incorrect?

Try `def test_meta(capsys: pytest.CaptureFixture[str]) -> None:`

**Review by mr-c — APPROVED** [review]

**Pull request merged**